### PR TITLE
Modify the pile-up rejection for HM analysis

### PR DESCRIPTION
**Pull request by JinjooSeo**
Modify the pile-up rejection for HM analysis

### CODE REVIEW

File: PWGHF/vertexingHF/AliAnalysisTaskSEXic0Semileptonic.cxx [hunk 1]
```diff
@@ -501,10 +538,50 @@ void AliAnalysisTaskSEXic0Semileptonic::UserExec(Option_t*)
...
+	Int_t nContributor = -999;
+	if (IsPA) nContributor = 5; //For pA, but there's a NOTE at above link - PLEASE CHECK
+	else
+	{
+		if      ( fNSPDTracklets < 20 ) nContributor = 3; //Conventional, valid for low multiplicity pp
+		else if ( fNSPDTracklets < 50 ) nContributor = 4; //20 ~ 50
+		else nContributor = 5; //>= 50, valid for high multiplicity pp and p-Pb
+	}
+	if (nContributor == -999) { cout <<"ERROR in pileup cut!\n"; assert(false); }
+	if (!IsMC && fEvt->IsPileupFromSPD(nContributor, 0.8, 3., 2., 5.)) return;
```
Comment (fgrosa) [inline PWGHF/vertexingHF/AliAnalysisTaskSEXic0Semileptonic.cxx:582]: We typically use the multi-vertexer algorithm for pileup rejection in pp (see https://github.com/alisw/AliPhysics/blob/b2904d50626090b88c2c5b445a7a0d758e7eb7ac/OADB/AliAnalysisUtils.h#L38), is there a reason for using the SPD based algorithm instead for this analysis?

Comment (JinjooSeo) [reply PWGHF/vertexingHF/AliAnalysisTaskSEXic0Semileptonic.cxx:582]: Hi @fgrosa, We used the SPD based rejection as this link (https://twiki.cern.ch/twiki/bin/viewauth/ALICE/AliDPGtoolsPileup). I missed the MV based rejection at that time. 
As you mentioned, I will replace the pileup rejection using MV based, and will check it how much difference will be shown.

**Comment by fgrosa** [comment]
+1

File: PWGHF/vertexingHF/AliAnalysisTaskSEXic0Semileptonic.cxx (hunk 1 above)
Comment (fgrosa) [reply PWGHF/vertexingHF/AliAnalysisTaskSEXic0Semileptonic.cxx]: Thanks a lot!

**Comment by fgrosa** [comment]
+test

**Comment by fgrosa** [comment]
+1

**Pull request merged**